### ISSUE TITLE
Linter rule implementation. Help wanted.

### ISSUE THREAD

**Issue by esnesarev** [Instagram/Fixit#174]
First of all – thank you so much for libCST and Fixit and making them both publicly available.

I'm currently trying to migrate my existing Pylint rules to Fixit and I'm stuck on one of them. If you can provide some help on how it can be implemented on libCST and Fixit – I would really appreciate it.

I want to make sure that builtin next() function is always called with an explicitly specified default value OR wrapped around with try / except and StopIteration exception is handled.

Invalid case:
```
def hello(things):
    next(thing for thing in things)
```
 
Valid cases:
```
def hello(things):
     next(thing for thing in things, "howdy")
...

def hello(things):
   try:
       return next((thing for thing in things))
   except StopIteration:
       return "hi"
```

My existing Pylint rule relies on [node_ignores_exception](https://github.com/PyCQA/pylint/blob/master/pylint/checkers/utils.py#L1006) helper.

p.s. If it's not an appropriate place to ask questions like that – please let me know where should I do that instead. Thanks!

**Comment by sumanth-v** [comment]
Hi can i share the link...

**Comment by amyreese** [comment]
Hey folks, we just released Fixit 2.0, which includes major changes to the core frameworks, linting engine, API, and CLI. In light of this new release, this issue is likely to be resolved or no longer relevant. Please take a look at the new [user guide](https://fixit.readthedocs.io/en/latest/guide.html#) or the [upgrade guide](https://fixit.readthedocs.io/en/latest/upgrade.html) for more information on how to migrate your projects to the new version.

If you believe this issue is still relevant or still occurring in the latest version, please feel free to open a new issue, or reply here with more details.

Thank you!
